_stbt/ocr.py

```diff
@@ -363,6 +391,57 @@ def match_text(text, frame=None, region=Region.ALL,
     return result
 
 
+# Python 2.7 & 3.6 have `re._pattern_type` but that will be removed in Python
+# 3.7 where they introduce `re.Pattern`.
+PatternType = type(re.compile(""))
+
+
+def apply_ocr_corrections(text, corrections=None):
+    """Applies the same corrections as `stbt.ocr`'s ``corrections`` parameter.
+
+    This is also available as a separate function, so that you can use it to
+    post-process old test artifacts using new corrections. See also
+    `stbt.set_global_ocr_corrections`.
+    """
+    if corrections:
+        text = _apply_ocr_corrections(text, corrections)
+    if global_ocr_corrections:
+        text = _apply_ocr_corrections(text, global_ocr_corrections)
+    return text
+
+
+def _apply_ocr_corrections(text, corrections):
+    # Match plain strings at word boundaries:
+    pattern = "|".join(r"\b(" + re.escape(k) + r")\b"
+                       for k in corrections
+                       if isinstance(k, basestring))
+    if pattern:
+        replace = lambda matchobj: corrections[matchobj.group(0)]
+        text = re.sub(pattern, replace, text)
+    # Match regexes:
+    for k, v in corrections.items():
+        if isinstance(k, PatternType):
+            text = re.sub(k, v, text)
+    return text
+
+
+global_ocr_corrections = {}
+
+
+def set_global_ocr_corrections(corrections):
+    """Specify default OCR corrections that apply to all calls to `stbt.ocr`
+    and `stbt.apply_ocr_corrections`.
+
+    See the ``corrections`` parameter of `stbt.ocr` for more details.
+
+    We recommend calling this function from ``tests/__init__.py`` to ensure it
+    is called before any test script is executed.
```

Comment:
I think the recommendation is good, but I'm not sure about the reasoning. I think the advantage of `__init__.py` is not when it's run, it's the fact that it's will be run for every test-case in your test-pack.  Maybe that's what's intended and I'm misreading putting too much emphasis on the word "before".